aries-site/src/examples/foundation/icons/IconCoreExample.js

```diff
@@ -138,11 +139,13 @@ export const IconControlsExample = () => (
     <CoreIcon
       Icon={Up}
       name="Collapse"
```

Comment:
is this `icon` still names Collapse I couldnt find it here? 
https://github.com/grommet/hpe-design-system/wiki/Migrating-from-grommet%E2%80%90icons-to-@hpe%E2%80%90design%5Cicons%E2%80%90grommet